data/brands/amenity/recycling.json

```diff
@@ -224,7 +224,7 @@
     {
       "displayName": "The Salvation Army",
       "id": "thesalvationarmy-f8d7ba",
-      "locationSet": {"include": ["us"]},
+      "locationSet": {"include": ["gb", "us"]},
```

Comment:
Should this mirror the charity shops:
https://github.com/osmlab/name-suggestion-index/blob/74b8e4e/data/brands/shop/charity.json#L516

Reply:
Raised in #5981 5981